### PR TITLE
Display sentieon dedup metrics in multiqc

### PR DESCRIPTION
**Pull request by asp8200**
Importing and displaying metrics from Sentieon-Dedup as "custom content" in the MultiQC report.

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: modules/nf-core/sentieon/dedup/main.nf [hunk 1]
```diff
@@ -60,6 +61,9 @@ process SENTIEON_DEDUP {
 
     sentieon driver $args $input_list -r ${fasta} --algo LocusCollector $args2 --fun score_info ${prefix}.score
     sentieon driver $args3 -t $task.cpus $input_list -r ${fasta} --algo Dedup $args4 --score_info ${prefix}.score --metrics ${metrics} ${prefix}${suffix}
+    # This following tsv-file is produced in order to get a proper tsv-file with Dedup-metrics for importing in MultiQC as "custom content".
+    # It should be removed once MultiQC has a module for displaying Dedup-metrics.
+    head -3 ${metrics} > ${metrics}.multiqc.tsv
```
Comment (maxulysse) [inline modules/nf-core/sentieon/dedup/main.nf:66]: I'm thinking of pushing that to modules actually, but that can wait

**Review by maxulysse — APPROVED** [review]

File: modules/nf-core/sentieon/dedup/main.nf (hunk 1 above)
Comment (asp8200) [reply modules/nf-core/sentieon/dedup/main.nf:66]: Sure, I can put it in the modules-repo if you prefer that.  Should I do that now?

Comment (maxulysse) [reply modules/nf-core/sentieon/dedup/main.nf:66]: Yeah, let's go for it

File: modules.json [hunk 2]
```diff
@@ -383,8 +383,9 @@
                     },
                     "sentieon/dedup": {
                         "branch": "master",
-                        "git_sha": "2cb2012b268b3502ac74463e510e76be99e2cdbf",
-                        "installed_by": ["modules"]
+                        "git_sha": "e311732e4d878d7e47872226bef5f8630dc81675",
+                        "installed_by": ["modules"],
+                        "patch": "modules/nf-core/sentieon/dedup/sentieon-dedup.diff"
```
Comment (maxulysse) [inline modules.json:388]:
```suggestion
                        "installed_by": ["modules"]

```

File: modules.json [hunk 3]
```diff
@@ -383,8 +383,9 @@
                     },
                     "sentieon/dedup": {
                         "branch": "master",
-                        "git_sha": "2cb2012b268b3502ac74463e510e76be99e2cdbf",
+                        "git_sha": "e311732e4d878d7e47872226bef5f8630dc81675",
                         "installed_by": ["modules"]
+
```
Comment (maxulysse) [inline modules.json:388]:
```suggestion
```

**Pull request merged**